setup.py

```diff
@@ -44,6 +44,7 @@
 ]
 
 dev_dependencies = [
+    "anyio",
```

Comment:
anyone else needs to get involved when adding new dep ?

Reply:
approving